src/test/resources/test-exception-3.kore.golden

```diff
@@ -1,6 +1,6 @@
-../src/test/resources//test-exception-3.kore:6:7:
+../src/test/resources//test-exception-3.kore:6:12:
   |
-6 |     'X:S []
-  |       ^
-unexpected ':'
-expecting '''
+6 |     "X:S []
+  |            ^
+unexpected newline
+expecting '"' or printable ASCII character
```

Comment:
Should we improve this error message? It could also be an escape sequence.

Reply:
well, it cannot be an escape sequence anymore, because it doesn't start with `\`.  The message changed because I used peekChar to look one char ahead to chose between escape chars and printable chars.

Reply:
It could be the start of an escape sequence, i.e. it could be `\` itself.

Reply:
that's what I'm saying, that there is a check for `\` before we get here, so it couldn't be  the start of an escape sequence.

Reply:
A `\` is permitted there by the syntax.

Reply:
I see. however, these messages are generated automatically from the "labels" we're adding, and because now we're not attempting `\` anymore in this context, it got "automagically" removed. Not sure how to put it back.

Reply:
That makes sense. I wouldn't worry about such a corner case right now.